### PR TITLE
[TCMN-57] docblock correction

### PR DESCRIPTION
**Pull request by cliffordp**
Back to being [CT-21] but not related to that at all.
~TCMN-57 (to be assigned)~

[CT-21]: https://moderntribe.atlassian.net/browse/CT-21

### CODE REVIEW

**Comment by cliffordp** [comment]
@bordoni, I got this email:
> Run failed: .github/workflows/phpcs.yml
> https://github.com/moderntribe/tribe-common/actions/runs/69610352

Any ideas?

**Comment by bordoni** [comment]
Just modified, `master` was still using the old name for the CI slug.

**Comment by bordoni** [comment]
That should go either against G20.04 or B20.04

**Comment by cliffordp** [comment]
@sc0ttkclark I created https://moderntribe.atlassian.net/browse/TCMN-57

File: src/Tribe/Template.php [hunk 1]
```diff
@@ -111,7 +111,7 @@ public function set_template_origin( $origin = null ) {
 			$this->origin = $origin;
 			$this->template_base_path = untrailingslashit( ! empty( $this->origin->plugin_path ) ? $this->origin->plugin_path : $this->origin->pluginPath );
 		} else {
-			$this->template_base_path = untrailingslashit( (array) explode( '/', $origin ) );
+			$this->template_base_path = untrailingslashit( (array) explode( '/', $origin ) ); // passing an array instead of a string?
```
Comment (cliffordp) [inline src/Tribe/Template.php:114]: @bordoni `untrailingslashit()` requires a String, not an Array. What _should_ be here?

Comment (bordoni) [reply src/Tribe/Template.php:114]: Invert the logic, first `untrailingslashit` to after `explode`.

Comment (bordoni) [inline src/Tribe/Template.php:114]:
```suggestion
			$this->template_base_path = (array) explode( '/', untrailingslashit( $origin ) );
```

**Pull request merged**